### PR TITLE
docs: model: scratch: Python code examples

### PR DESCRIPTION
**Pull request by sk-ip**
Python code examples for using dffml_model_scratch.
references #434

### CODE REVIEW

**Comment by pdxjohnny** [comment]
Nice! We need to make sure to have the examples be tested too though, similar to https://github.com/intel/dffml/commit/e7aaeb1e1e2f1b2ce74d1fe6ecc6422bf6df178c

**Comment by sk-ip** [comment]
@pdxjohnny , this is ready to be reviewed.

One thing I found is that when we chage the docstrings to something like https://github.com/intel/dffml/pull/541/commits/14ca8b3ab793f3a82ed6309b3d3ce5c27c1004bd to make cli code and python code testable, the docstrings looks weird when trying to refer the docstrings it in some other program.
This is happening in `dffml_model_tensorflow` as well

File: model/scratch/examples/scratchlgrsag/dataset.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+cat > dataset.csv << EOF
+  f1,ans
```
Comment (pdxjohnny) [inline model/scratch/examples/scratchlgrsag/dataset.sh:2]:
```suggestion
f1,ans
```

File: model/scratch/examples/scratchlgrsag/dataset.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  0.1,0
```
Comment (pdxjohnny) [inline model/scratch/examples/scratchlgrsag/dataset.sh:3]:
```suggestion
0.1,0
```

File: model/scratch/examples/scratchlgrsag/dataset.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  0.7,1
```
Comment (pdxjohnny) [inline model/scratch/examples/scratchlgrsag/dataset.sh:4]:
```suggestion
0.7,1
```

File: model/scratch/examples/scratchlgrsag/dataset.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+  0.6,1
```
Comment (pdxjohnny) [inline model/scratch/examples/scratchlgrsag/dataset.sh:5]:
```suggestion
0.6,1
```

File: model/scratch/examples/scratchlgrsag/dataset.sh [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+  0.2,0
```
Comment (pdxjohnny) [inline model/scratch/examples/scratchlgrsag/dataset.sh:6]:
```suggestion
0.2,0
```

File: model/scratch/examples/scratchlgrsag/dataset.sh [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+  0.8,1
```
Comment (pdxjohnny) [inline model/scratch/examples/scratchlgrsag/dataset.sh:7]:
```suggestion
0.8,1
```

File: model/scratch/examples/scratchlgrsag/test_scratchlgrsag.py [hunk 7]
```diff
@@ -0,0 +1,58 @@
+import os
```
Comment (pdxjohnny) [inline model/scratch/examples/scratchlgrsag/test_scratchlgrsag.py:1]: We need an `__init__.py` in `model/scratch/examples/scratchlgrsag/`

```console
$ touch model/scratch/examples/scratchlgrsag/__init__.py
```

Comment (sk-ip) [reply model/scratch/examples/scratchlgrsag/test_scratchlgrsag.py:1]: done

**Comment by pdxjohnny** [comment]
Great work!! Thanks @sk-ip! :D

**Pull request merged**